apps/acrs/translations/src/en/validation.json

```diff
@@ -15,6 +15,14 @@
     "confirm-referrer-email": {
         "required": "You must select an option"
     },
+    "referrer-email-options": {
+        "required": "Tell us if the referrer has an email address"
+    },
+    "referrer-email-address": {
+      "maxlength": "Email address must be between 6 and 254 characters",
+      "email": "Enter a real email address",
```

Comment:
I think 'valid' is better than 'real'

Reply:
These validation strings are from the ['ACRS validation messages' doc](https://homeofficegovuk.sharepoint.com/:x:/s/hof-team/ESROuM_igt9HsFaFLQVkwPcBI9ad0H9hjxdFis3gVxIiig?e=FNd9T3)